tests/schema/extensions/test_extensions.py

```diff
@@ -541,3 +542,44 @@ def ping(self) -> str:
     assert result.errors == [GraphQLError("Well you asked for it")]
 
     assert mock_original_execute.call_count == 1
+
+
+def test_extend_error_format_example():
+    # Test that the example of how to extend error format
+
+    class ExtendErrorFormat(Extension):
+        def on_request_end(self):
+            result = self.execution_context.result
+            if hasattr(result, "errors") and result.errors is not None:
+                processed_errors = []
+                for error in result.errors:
+                    processed_errors.append(
+                        StrawberryGraphQLError(
+                            extensions={"additional_key": "additional_value"},
+                            nodes=error.nodes,
+                            source=error.source,
+                            positions=error.positions,
+                            path=error.path,
+                            original_error=error.original_error,
+                            message=error.message,
+                        )
+                    )
+                result.errors = processed_errors
```

Comment:
Can't resist proposing the functional version (sorry, heard a lot about it benefits in university lately 🫠)

```suggestion
                result.errors = [
                    StrawberryGraphQLError(
                        extensions={"additional_key": "additional_value"},
                        nodes=error.nodes,
                        source=error.source,
                        positions=error.positions,
                        path=error.path,
                        original_error=error.original_error,
                        message=error.message,
                    )
                    for error in result.errors
                ]
```

That's also just an suggestion :)

Reply:
That's nice 😆